tests/Order/OrderNumberGeneratorTest.php

```diff
@@ -37,10 +38,10 @@ public function usesConfiguredOffsetAndModelCount()
         config(['cashier.order_number_generator.offset' => 15]);
         $this->generator = new OrderNumberGenerator;
 
-        $this->assertTrue(ends_with($this->generator->generate(), '16'));
+        $this->assertTrue(Str::endsWith($this->generator->generate(), '16'));
 
         factory(Order::class, 3)->create();
-        $this->assertTrue(ends_with($this->generator->generate(), '19'));
+        $this->assertTrue(Str::endsWith($this->generator->generate(), '19'));
 
     }
 
```

Comment:
Please revert to the original string helpers (because of backwards compatibility). #52 is solving this.